src/AWS.Deploy.CLI/Commands/TypeHints/BeanstalkEnvironmentCommand.cs

```diff
@@ -20,13 +24,24 @@ public BeanstalkEnvironmentCommand(IAWSResourceQueryer awsResourceQueryer, ICons
             _consoleUtilities = consoleUtilities;
         }
 
-        public async Task<object> Execute(Recommendation recommendation, OptionSettingItem optionSetting)
+        private async Task<T> GetData<T>(Recommendation recommendation, OptionSettingItem optionSetting)
         {
-            var currentValue = recommendation.GetOptionSettingValue(optionSetting);
             var applicationOptionSetting = recommendation.GetOptionSetting(optionSetting.ParentSettingId);
-
             var applicationName = recommendation.GetOptionSettingValue(applicationOptionSetting) as string;
             var environments = await _awsResourceQueryer.ListOfElasticBeanstalkEnvironments(applicationName);
+            return JsonConvert.DeserializeObject<T>(JsonConvert.SerializeObject(environments));
```

Comment:
This looks very silly to serialize and then immediately deserialize the object. Especially since I'm pretty sure we are always doing this to the same type of `EnvironmentDescription`. I think you are copying and pasting this pattern to all of the type hint commands but it looks really weird and we should fix this up.